modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerTest.java

```diff
@@ -449,6 +452,8 @@ public void tableManagerStopTest() throws Exception {
 
         TableManager tableManager = tblManagerFut.join();
 
+        doThrow(new NodeStoppingException()).when(rm).stopRaftGroup(any());
```

Comment:
- And what about checking exception handling logic inside org.apache.ignite.internal.util.IgniteUtils#runAll?
- And what about checking exception handling logic during storage stop?

Reply:
I removed IgniteUtils#runAll.
Added new tests.